### PR TITLE
Add visionexpress_gb spider (544 optician stores)

### PR DESCRIPTION
**Pull request by mjoe999**
Required some fixes to the Google URL parse code.

@Cj-Malone : fixes above are for when query parameter required was first in the param list on the link!

### CODE REVIEW

File: locations/spiders/visionexpress_gb.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+def extract_google_position(item, response):
+    for link in response.xpath("//img/@src").extract():
+        if link.startswith("https://maps.googleapis.com/maps/api/staticmap"):
+            item["lat"], item["lon"] = url_to_coords(link)
+            return
+
+
+def set_located_in(item, located_in, located_in_wikidata):
```
Comment (Cj-Malone) [inline locations/spiders/visionexpress_gb.py:14]: This is now in 2 spiders, should it be moved to `items.py`?

**Comment by Cj-Malone** [comment]
I've checked the new `url_to_coords`, but assuming you ran `pytest` and everything still works, LGTM.

File: locations/spiders/visionexpress_gb.py (hunk 1 above)
Comment (mjoe999) [reply locations/spiders/visionexpress_gb.py:14]: It's on my list but not for this commit.

**Comment by mjoe999** [comment]
Fixes #3676

File: locations/spiders/visionexpress_gb.py (hunk 1 above)
Comment (iandees) [reply locations/spiders/visionexpress_gb.py:14]: I still don't understand this pattern. It doesn't make sense to extract two assignments into a function. Is there extra functionality you're thinking would go in here in the future?

I could see an argument to be made for better readability, but the misdirection makes it harder to follow than putting the assignments in line in my opinion.

**Pull request merged**